internal/scheduler/legacyscheduler_test.go

```diff
@@ -1566,6 +1596,54 @@ func TestReschedule(t *testing.T) {
 			// To test the gang evictor, we need to disable stochastic eviction.
 			NodeEvictionProbability: pointer.Pointer(0.0),
 		},
+		"gang preemption avoid cascading preemptions": {
```

Comment:
Test to ensure evicting the rest of a gang job doesn't cause more partial gang evictions.